packages/server/src/utilities/fileSystem/index.js

```diff
@@ -109,21 +110,34 @@ exports.apiFileReturn = contents => {
   return fs.createReadStream(path)
 }
 
+exports.defineFilter = includeRows => {
```

Comment:
Not one for now - but think a constants file with different export mechanisms could be good here - as we will likely add more in future and we would have to keep extending this conditional logic